src/data/screenshots_de.json

```diff
@@ -24,190 +25,190 @@
-                     { "filename": "/assets/screenshots/2-10/de/android/PrintVaccinationFragment_3", "alt": "Impfzertifikat drucken Bild 3 von 4" },
-                     { "filename": "/assets/screenshots/2-10/de/android/PrintVaccinationFragment_4", "alt": "Impfzertifikat drucken Bild 4 von 4" }
+                     { "filename": "/assets/screenshots/2-11/de/android/PrintVaccinationFragment", "alt": "Impfzertifikat drucken Bild 1 von 4" },
+                     { "filename": "/assets/screenshots/2-11/de/android/PrintVaccinationFragment_2", "alt": "Impfzertifikat drucken Bild 2 von 4" },
+                     { "filename": "/assets/screenshots/2-11/de/android/PrintVaccinationFragment_3", "alt": "Impfzertifikat drucken Bild 3 von 4" },
+                     { "filename": "/assets/screenshots/2-11/de/android/PrintVaccinationFragment_4", "alt": "Impfzertifikat drucken Bild 4 von 4" }
                  ]
              },
               {
                   "title": "Genesenenzertifikat",
                   "anchor": "android_recovery_certificate",
                   "images": [
-                    { "filename": "/assets/screenshots/2-10/de/android/RecoveryCertificateDetailsFragment_recovered", "alt": "Genesenenzertifikat Bild 1 von 2" },
-                    { "filename": "/assets/screenshots/2-10/de/android/RecoveryCertificateDetailsFragment_recovered_2", "alt": "Genesenenzertifikat Bild 2 von 2" }
+                    { "filename": "/assets/screenshots/2-11/de/android/RecoveryCertificateDetailsFragment_recovered", "alt": "Genesenenzertifikat Bild 1 von 2" },
+                    { "filename": "/assets/screenshots/2-11/de/android/RecoveryCertificateDetailsFragment_recovered_2", "alt": "Genesenenzertifikat Bild 2 von 2" }
                         ]
               },
               {
                   "title": "Zertifikate vor einer Reise auf Gültigkeit prüfen",
                   "anchor": "android_check_validity_certificate",
                   "images": [
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_vaccination_certificate_valid_details_part3", "alt": "Zertifikate auf Gültigkeit prüfen Bild 1 von 4" },
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_certificate_validation_date_selection", "alt": "Zertifikate auf Gültigkeit prüfen Bild 2 von 4" },
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_certificate_validation_country_selection", "alt": "Zertifikate auf Gültigkeit prüfen Bild 3 von 4" },
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_certificate_validation", "alt": "Zertifikate auf Gültigkeit prüfen Bild 4 von 4" }
+                      { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_vaccination_certificate_valid_details_part3", "alt": "Zertifikate auf Gültigkeit prüfen Bild 1 von 4" },
```

Comment:
This iOS screenshot has been taken on version 2.10 and not on version 2.11. The info text reg. CovPassCheck is missing.

It should be replaced with a screenshot taken on version 2.11.

Reply:
@Ein-Tim Thanks added!